### ISSUE TITLE
Document permission for connection prompts in MacOS Big Sur

### ISSUE THREAD

**Issue by bmann** [fission-suite/fission#459]
When running fission setup and fission register / publish, Big Sur will prompt to allow connections for fission-cli.

The CLI will never complete, so you need to approve the prompt, Ctl-C quit the current run, and then run it again after approving the connection prompts, and then it will work.

Thanks @edalzell

**Comment by edalzell** [comment]
There should be a way to have it work w/ no user interaction (like re-running), cuz dnsmasq and Nginx prompt (like when you install Laravel's `valet` and then it "just works". No clue how they do it though.

**Comment by bmann** [comment]
@edalzell yeah it will likely have to do with code signing. Keeping this here to document / investigate.
